freqtrade/exchange/exchange.py

```diff
@@ -607,7 +608,7 @@ def cancel_order(self, order_id: str, pair: str) -> None:
         try:
             return self._api.cancel_order(order_id, pair)
         except ccxt.InvalidOrder as e:
-            raise DependencyException(
+            raise InvalidOrderException(
```

Comment:
any particular reason why you don't retry cancelling if this error occurs - but you still retry `get_order()` on the same error?

Reply:
you are right if you consider consistency. but get_order (in the context of this PR) never returns InvalidOrderException. but I am going to add it.

Reply:
oh no, sorry forget the above comment ! in fact I myself also use DependencyException for get_order